lib/crowbar/init/application.rb

```diff
@@ -315,6 +315,9 @@ class Application < Sinatra::Base
       # api_version "2.0"
       post "/api/upgrade/new" do
         api_constraint(2.0)
+        upgrade_status = ::Crowbar::UpgradeStatus.new(logger)
+        upgrade_status.start_step
```

Comment:
Please change to upgrade_status.start_step :database once crowbar/crowbar-core#816 is merged